app/services/db/add_document_verification_and_selfie_costs.rb

```diff
@@ -1,7 +1,8 @@
 module Db
   class AddDocumentVerificationAndSelfieCosts
-    def initialize(user_id:, service_provider:)
+    def initialize(user_id:, service_provider:, liveness_checking_enabled:)
       @service_provider = service_provider
+      @liveness_checking_enabled = liveness_checking_enabled
```

Comment:
@eileen-nava , yes that was originally there and removed, since we may have planned changes for cost accounting due to profiles support changes.

Reply:
> I want to be sure I grok this file.
> 
> Does this do costing for backend calls to docauth vendors regardless of which vendor receives the request? Also, do we need to call `add_cost` for the selfie image?

👀 @zachmargolis ?

Reply:
@amirbey , we record cost as a database entry for `selfie` ( past implementaion), with this function: (in our case, the token would be `selfie`)

https://github.com/18F/identity-idp/blob/4ae920a42852754fec1da8f110393845184e9ae9/app/services/db/sp_cost/add_sp_cost.rb#L18-L37

There is no differentiation of SP per se, but have no knowledge how we charge client based on this information.

If we adding this now, this would assume downstream billing process can process it, at least without errors which I don't know yet.